### ISSUE TITLE
store.test.js is duplicated

### ISSUE THREAD

**Issue by akaRem** [mxstbr/react-boilerplate#631]
subj test suite is located in 2 places with just import difference
```
$ wc -l app/store.test.js
      27 app/store.test.js
$ diff app/store.test.js app/tests/store.test.js
6c6
< import configureStore from './store';
---
> import configureStore from '../store';
```

**Comment by mxstbr** [comment]
Is that after cleaning or in the example app?

(PR? :grin:)

**Comment by akaRem** [comment]
@mxstbr Noticed after clean command.. And looks like it didn't exit before that

**Comment by mxstbr** [comment]
Fixed by the amazing @uzarubin in #858!
